src/main/java/com/google/devtools/build/lib/starlarkbuildapi/android/AndroidConfigurationApi.java

```diff
@@ -94,13 +94,6 @@ public interface AndroidConfigurationApi extends StarlarkValue {
       documented = false)
   ImmutableList<String> getTargetDexoptsThatPreventIncrementalDexing();
 
-  @StarlarkMethod(
-      name = "use_workers_with_dexbuilder",
```

Comment:
Let's cherry-pick your GraveyardOptions commit into this PR as well for completeness.

https://github.com/bazelbuild/bazel/pull/17351/commits/9bfc2b7548c65e1bc4b0f0273549f07bfb444cca